sources/api/README.md

```diff
@@ -28,6 +28,23 @@ It's available in Bottlerocket, whether you're accessing it through a control ch
 
 Rust code can use the `apiclient` library to make requests to the Unix-domain socket of the [apiserver](#apiserver).
 
+The API socket can be mounted and accessed from a privileged container.
+An example pod specification to expose API access would be similar to:
```

Comment:
It would be good to link to the security guidance on this topic - https://github.com/bottlerocket-os/bottlerocket/blob/develop/SECURITY_GUIDANCE.md#restrict-access-to-the-host-api-socket